### PR TITLE
Initial pass at space type predictions

### PR DESCRIPTION
**Pull request by bradcline-kc**

### CODE REVIEW

**Comment by JacksonJ-KC** [comment]
changing this to a draft because it doesn't appear to be ready for review

File: interface/rulesets/ashrae9012019prm/views/spaces.py [hunk 1]
```diff
@@ -244,3 +460,147 @@ def add_row(self, i, space_name):
...
+                # Get area, both LPDs, and lighting power for the space
+                space_area = space.try_float(space.get_inp(BDL_SpaceKeywords.AREA))
+                space_1_lpd = SPACE_TYPE_LPD.get(
+                    SPACE_TYPE_CODES.get(space_type_codes[0]), None
+                )
+                space_2_lpd = SPACE_TYPE_LPD.get(
+                    SPACE_TYPE_CODES.get(space_type_codes[1]), None
+                )
+                # TODO: See notes below in calculate_space_areas(). Lighting power is off.
+                space_lighting_power = space_area * space.try_float(
```
Comment (JacksonJ-KC) [inline interface/rulesets/ashrae9012019prm/views/spaces.py:389]: You should reuse the calculated values stored in`Space.int_ltg_power_per_area` because the modeled lighting power can be a combination of `LIGHTING_W_AREA` & `LIGHTING_KW` and this has already been accounted for here

**Comment by bradcline-kc** [comment]
Tasks left here:

- [x] Edge case - lpd1 == lpd2
- [x] Edge case - test blank activity desc
- [x] Test large model
- [x] Populate "red box" data

**Comment by bradcline-kc** [comment]
Probably separate PR, but I need to add unit tests for this. Need to come up with a good framework for doing unit tests for the GUI. We'll have to instantiate a lot of stuff (app data, views, etc.) to allow the tests to run, I think. I'm wondering now if we should pull the space type predictions from the Spaces View? If we do that, we can build unit tests without having to make a SpaceView or SpaceSubview.

**Comment by bradcline-kc** [comment]
@JacksonJ-KC need to chat about how to split up misc_eq_ids and int_ltg_ids if there are multiple. Would _n_ number of IDs indicate _n_ number of spaces? I'd imagine it's more complicated than that.

**Comment by JacksonJ-KC** [comment]
> @JacksonJ-KC need to chat about how to split up misc_eq_ids and int_ltg_ids if there are multiple. Would _n_ number of IDs indicate _n_ number of spaces? I'd imagine it's more complicated than that.

I think we need to split all of the misc eq and int ltg from the original modeled space to each of the split spaces. If the original space has 2 misc eq and 3 int ltg and is split into 3 spaces then all 3 spaces have 2 misc eq and 3 int ltg

**Comment by JacksonJ-KC** [comment]
@bradcline-kc please address merge conflicts and update if necessary

**Comment by bradcline-kc** [comment]
Resolved
